packages/contract-wrappers/src/utils/decorators.ts

```diff
@@ -1,8 +1,31 @@
 import * as _ from 'lodash';
 
-import { AsyncMethod, ContractWrappersError, SyncMethod } from '../types';
-
-import { constants } from './constants';
+export enum ContractWrappersError {
```

Comment:
Can we get rid of this now or should we wait for V3 better revert reasons? This file mainly converts Ethereum error msgs (e.g. 'out of gas') into these standardised error codes.

Reply:
Let's keep it for now.